c2_utils/src/mfx_dev_va.cpp

```diff
@@ -200,12 +200,16 @@ mfxStatus MfxDevVa::InitMfxSession(MFXVideoSession* session)
 std::shared_ptr<MfxFrameAllocator> MfxDevVa::GetFrameAllocator()
 {
     MFX_DEBUG_TRACE_FUNC;
+    if (m_usage == Usage::Encoder && isDedicated())
```

Comment:
Can you explain why we need the dGPU info for encoder only?

Thanks,
Tianmi

Reply:
va_allocator only work for encoder, for decoder, we use va_pool_allocator